ft/commands.go

```diff
@@ -274,6 +276,36 @@ func showVersion(data *CmdArgs) error {
 	return nil
 }
 
+func showDirectoryVar(data *CmdArgs) error {
+	rt := runtime.GOOS
+	var dir string
+	switch rt {
+	case "linux":
+		cmd := exec.Command("pwd")
+		d, err := cmd.Output()
+		dir = strings.Trim(string(d), "\n")
+		if err != nil {
+			return err
+		}
+	default:
+		fmt.Println("This command is not supported on your os")
+	}
+
+	pathMaps := data.allPaths
+
+	for k := range pathMaps {
+		v, _ := pathMaps[k]
+
+		if strings.Compare(v, dir) == 0 {
+			fmt.Println(strings.Trim(k, "\n"))
```

Comment:
I'd rather this print something clear for the user like "[path] is saved to key [name of key]".

Reply:
```sh
Directory /home/go/fastTravelCLI is saved to key : fast
```

I have changed output string to this.